### ISSUE TITLE
basic documentation

### ISSUE THREAD

**Issue by patroqueeet** [Execut3/django-discount#1]
would be nice though.... with install, setup, config, short description what it does and how it works

**Comment by Execut3** [comment]
yes of course. really appreciate your comment, it's not final yet. Will update docs, setups, and all needed items in the git repo.
